src/components/FolioImageModal.vue

```diff
@@ -1,13 +1,105 @@
 <template>
-  <div />
+  <transition name="modal">
+    <div
+      class="modal-mask fixed top-0 left-0 w-100 h-100 dt bg-black-70"
+      @click="$emit('close')"
+    >
+      <div
+        class="dtc v-mid"
+      >
+        <div class="modal-container w-80 w-40-ns pa1 bg-black br3 shadow-2 helvetica center">
+          <div class="modal-header mt0">
+            <slot
+              name="header"
+              class=""
+            >
```

Comment:
the button shouldn't stay in the header as the header is overwritable and you might lose the ability to close the modal